util/envDynawo.sh

```diff
@@ -952,6 +954,34 @@ build_dynawo() {
   return ${RETURN_CODE}
 }
 
+build_dynaflow() {
+  if [ ! -d "$DYNAWO_BUILD_DIR" ]; then
+    error_exit "$DYNAWO_BUILD_DIR does not exist."
+  fi
+  cd $DYNAWO_BUILD_DIR
+  config_dynawo || error_exit "Error during config_dynawo."
+  build_dynawo_core || error_exit "Error during build_dynawo_core."
+  build_dynawo_models_cpp || error_exit "Error during build_dynawo_models_cpp."
+  build_dynawo_target DYNAFLOW_MODELS || error_exit "Error during build_dynawo_models."
+  build_dynawo_solvers
+  RETURN_CODE=$?
+  return ${RETURN_CODE}
+}
+
+build_dynaswing() {
+  if [ ! -d "$DYNAWO_BUILD_DIR" ]; then
+    error_exit "$DYNAWO_BUILD_DIR does not exist."
+  fi
+  cd $DYNAWO_BUILD_DIR
+  config_dynawo || error_exit "Error during config_dynawo."
+  build_dynawo_core || error_exit "Error during build_dynawo_core."
+  build_dynawo_models_cpp || error_exit "Error during build_dynawo_models_cpp."
+  build_dynawo_target DYNASWING_MODELS || error_exit "Error during build_dynawo_models."
```

Comment:
Could we avoid duplicated code and create 1 function with an argument?

Reply:
I will do it

Reply:
done